include/packager/live_packager_export.h

```diff
@@ -128,6 +128,25 @@ LivePackagerStatus_t livepackager_package_timedtext(LivePackager_t lp,
                                                     size_t seg_len,
                                                     LivePackagerBuffer_t dest);
 
+
+//
+// Logging
+//
+typedef enum LogSeverity {
+  INFO = 0,
+  WARNING = 1,
+  ERROR = 2,
+  FATAL = 3,
+} LogSeverity_t;
+
+void lp_initializeLog(LogSeverity_t sev);
+void lp_installCustomLogSink();
```

Comment:
If we don't install the custom sink will we get the current behavior? Maybe add some additional comments.